Assets/Scripts/Core/Tilemap.cs

```diff
@@ -142,21 +151,15 @@ public class SaveObject {
 
 			public void SetNodeSprite(NodeSprite nodeSprite) {
 				this.nodeSprite_ = nodeSprite;
-				if(nodeSprite == NodeSprite.NONE) {
-					// Should be error state!
-					Debug.Log("NodeSprite none will become an hard error in the future!");
-					this.walkable = true;
-					this.hitChanceModifier = floorHitChanceModifier;
-					this.cover = false;
-				} else if(nodeSprite == NodeSprite.PIT) {
+				if(nodeSprite == NodeSprite.PIT) {
 					this.walkable = false;
-					this.hitChanceModifier = pitChanceModifier;
+					this.hitChanceModifier = pitHitChanceModifier;
 					this.cover = false;
 				} else if(nodeSprite == NodeSprite.FLOOR) {
 					this.walkable = true;
 					this.hitChanceModifier = floorHitChanceModifier;
 					this.cover = false;
-				} else if(nodeSprite == NodeSprite.WALL) {
+				} else if(nodeSprite == NodeSprite.WALL_FULL_TEXTURE || nodeSprite == NodeSprite.WALL_BACK_TEXTURE || nodeSprite == NodeSprite.WALL_FULL_MIDDLE_TEXTURE || nodeSprite == NodeSprite.NONE || nodeSprite == NodeSprite.DOOR_TOP || nodeSprite == NodeSprite.DOOR_MIDDLE || nodeSprite == NodeSprite.DOOR_BOTTOM) {
```

Comment:
A bit of a long OR chain, maybe change this to multiple lines for readability.

Reply:
Yep, not wrong, I'll split it up.